pkgs/tools/misc/nvimpager/default.nix

```diff
@@ -0,0 +1,48 @@
+{ fetchFromGitHub
+, pkgs
+}:
+
+pkgs.stdenv.mkDerivation rec {
+  pname = "nvimpager";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "lucc";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1xy5387szfw0bp8dr7d4z33wd4xva7q219rvz8gc0vvv1vsy73va";
+  };
+
+  buildInputs = with pkgs; [
+    ncurses # for tput
+    neovim
```

Comment:
what if I use a custom neovim ? maybe it should just be wrapped so that using a different neovim doesn't trigger a rebuild though I would think it's a pretty small package.

Reply:
Sorry I don't understand the question because I know to little about nix. I assume that this will use `pkgs.nvimpager` so if you  use an overlay I think it should use your overlayed neovim (unless I understand overlays wrongly).

Or are you asking why I did not put `nvimpager` into the argument set of the package? If so: no reason, just ignorance to the benifits. (please enlighten me :)

What should be wrapped how?

Reply:
I don't think that's necessary: we don't even compile something here so rebuilds are cheap.

Reply:
It causes a rebuild because the full path to neovim is hard coded into the shell script (the shell script accepts `$NVIM` to override the neovim executable at runtime and I set the full path in `preBuild` hook. I could also leave the default value at "nvim" and add the neovim derivation to the `buildInputs`)